app/models/Level.js

```diff
@@ -651,6 +651,21 @@ module.exports = (Level = (function () {
         })
       })
     }
+
+    getClassroomSub (teacher) {
```

Comment:
can we rename it to hasPermission or hasAccess and just provide whether the user should be able to access the level or not as the goal of this function?